ansible/roles/redpanda_broker/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+# installs necessary dependencies for running redpanda
+- name: Install dependencies
+  ansible.builtin.include_tasks: install-node-deps.yml
+  when: prep_deps | default(false) | bool
```

Comment:
This should be true by default. Users should opt-out of this rather than having to opt-in.